.github/workflows/test.yaml

```diff
@@ -9,6 +9,22 @@ on:
     branches: [ develop, develop-0.1, develop-0.2 ]
 
 jobs:
+  cargotest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions-rs/toolchain@v1
```

Comment:
Both of these `actions-rs/*` actions have had their repositories archived late last year. Probably fine for now but it would be worth thinking about alternatives in the near future

Reply:
Thanks for the heads up, was not aware of this, better to use maintained actions, updated